docs/website/docs/deployment-configurations/gpu-vulkan.md

```diff
@@ -74,10 +74,11 @@ so it can execute the model on GPU via Vulkan.
 
 #### Build runtime from source
 
-Please make sure you have followed the [Getting started][get-started] page
-to build IREE for Linux/Windows and the [Android cross-compilation][android-cc]
-page for Android. The Vulkan HAL driver is compiled in by default on non-Apple
-platforms.
+Please make sure you have followed the
+[Getting started](../building-from-source/getting-started.md) page to build IREE
+for Linux/Windows and the
+[Android cross-compilation](../building-from-source/android.md) page for
+Android. The Vulkan HAL driver is compiled in by default on non-Apple platforms.
 
 <!-- TODO(??): a way to verify Vulkan is compiled in and supported -->
```

Comment:
We could add these commands to these pages too:

```
iree-compile --iree-hal-list-target-backends
iree-run-module --dump_devices
```

(Trying to keep this PR focused, but these docs could use a refresh)